includes/common.php

```diff
@@ -1643,7 +1643,8 @@ function set_numeric($value, $default = 0)
 
 function check_git_exists()
 {
-    if (`which git`) {
+    exec('git >> /dev/null 2>&1', $response, $exit_code);
```

Comment:
Technically no but we don't care for the output at all so I though suppressing it would be better. The exit code is all we care about here.